modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/common/CustomThemeDeployerTest.java

```diff
@@ -0,0 +1,96 @@
+package org.wso2.ds.ui.integration.test.common;
+
+import org.openqa.selenium.By;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.Test;
+import org.wso2.carbon.automation.engine.frameworkutils.FrameworkPathUtil;
+import org.wso2.ds.ui.integration.util.DSUIIntegrationTest;
+
+import javax.xml.xpath.XPathExpressionException;
+import java.io.File;
+import java.net.MalformedURLException;
+
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertTrue;
+
+/**
+ * Test cases related to deploying a custom theme files using a carbon archive file
+ */
+public class CustomThemeDeployerTest extends DSUIIntegrationTest {
+    private static final String USER_NAME = "admin";
+    private static final String PASSWORD = "admin";
+    private static final String CUSTOM_THEME_NAME = "custom-theme-sample";
+
+    /**
+     * Initializes class
+     */
+    public CustomThemeDeployerTest() {
+        super();
+    }
+
+    /**
+     * This test case checks for the availability of custom theme files before deploying, after deploying and after
+     * un deploying the .car file
+     *
+     * @throws MalformedURLException
+     * @throws XPathExpressionException
+     * @throws InterruptedException
+     */
+    @Test(groups = "wso2.ds.common", description = "Deploying and un-deploying a custom theme in the admin console")
+    public void addDeleteCustomTheme() throws MalformedURLException, XPathExpressionException, InterruptedException {
```

Comment:
better throwing Exception class in test method rather than throwing each exceptions.